packages/blockchain-wallet-v4-frontend/src/scenes/Transactions/index.tsx

```diff
@@ -319,6 +321,7 @@ class TransactionsContainer extends React.PureComponent<Props> {
                 sourceType={sourceType}
               />
             ))}
+          {isCustodialCoin && <RiskInvestment coin={coin} />}
```

Comment:
Let's check with Seba if we actually need to check for whether coin is custodial or not. Separate between DeFi and Custodial is clear in superapp but not in v4. So maybe we should be showing the message to UK users all the time on this page.

Also `isCustodialCoin` does not seem to include `BTC`.

Reply:
Also this component has to be shown only to UK users.

Reply:
I don't understand